docs/src/content/docs/book/integers.mdx

```diff
@@ -158,6 +158,47 @@ struct Scrooge {
 }
 ```
 
+### Variable integer types {#serialization-varint}
```

Comment:
This needs to be merged with the `coins` description, since it's just an alias to `varuint16`